modules/nw-udn-additional-config-details.adoc

```diff
@@ -37,8 +40,12 @@ When enabled, OVN-Kubernetes applies the IP configuration to the SDN infrastruct
 **Disabled:** +
 When disabled, OVN-Kubernetes only assigns MAC addresses and provides layer 2 communication, which allows users to configure IP addresses. `Disabled` is only available for layer 2 (secondary) networks. By disabling IPAM, features that rely on selecting pods by IP, for example, network policy, services, and so on, no longer function. Additionally, IP port security is also disabled for interfaces attached to this network. The `subnets` field must be empty when `spec.ipam.mode` is set to `Disabled.`
 
-|`spec.layer2.mtu` and `spec.layer3.mtu`
+|`spec.network.<topology>.mtu`
+|`spec.<topology>.mtu`
 |integer
 |The maximum transmission units (MTU). The default value is `1400`. The boundary for IPv4 is `576`, and for IPv6 it is `1280`.
+|====
+
+where:
 
-|====
\ No newline at end of file
+`<topology>`:: Is one of `layer2` or `layer3`.
```

Comment:
@JoeAldinger As in the table topology is not specified as layer2 or layer3 but it has CUDN column as well.
Include localnet for 4.19
Keep layer2 and layer3 for prior versions

Reply:
Yes, this needs to go in prior to me fixing the table topology for 4.19. That is why it is a blocker for me.